arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/halt/SelfdestructSection.java

```diff
@@ -265,11 +264,16 @@ public void resolvePostTransaction(
             .findFirst()
             .orElseThrow(() -> new IllegalStateException("Account not found"));
 
-    if (hubStamp == hubStampOfTheSelfDestructThatSealedTheDeal) {
+    if (hubStamp == hubStampOfTheActionableSelfDestruct) {
       selfdestructScenarioFragment.setScenario(
           SelfdestructScenarioFragment.SelfdestructScenario
               .SELFDESTRUCT_WONT_REVERT_NOT_YET_MARKED);
 
+      hub.transients()
+          .conflation()
+          .deploymentInfo()
+          .deploymentUpdateForSuccessfulSelfDestruct(selfdestructorAccountBefore.address());
```

Comment:
We do the `DeploymentInfo` update for successful, unreverted and `NOT_YET_MARKED` SELFDESTRUCT's here.